source/vstd/Cargo.toml

```diff
@@ -21,3 +21,12 @@ state_machines_macros = { path = "../state_machines_macros" }
 default = ["std"]
 std = ["alloc"]
 alloc = []
+allocator = []
+strict_provenance_atomic_ptr = []
```

Comment:
Can you explain these two additions?

Reply:
Without declaring `allocator` and `strict_provenance_atomic_ptr` as features in `Cargo.toml`, `cargo build` complains:

```
warning: unexpected `cfg` condition value: `allocator`
  --> /home/nickolai/refsrc/verus/source/vstd/vstd.rs:12:35
   |
12 | #![cfg_attr(any(verus_keep_ghost, feature = "allocator"), feature(allocator_api))]
   |                                   ^^^^^^^^^^^^^^^^^^^^^
   |
   = note: expected values for `feature` are: `alloc`, `default`, and `std`
   = help: consider adding `allocator` as a feature in `Cargo.toml`
   = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration
   = note: requested on the command line with `-W unexpected-cfgs`

warning: unexpected `cfg` condition value: `strict_provenance_atomic_ptr`
   --> /home/nickolai/refsrc/verus/source/vstd/atomic.rs:670:33
    |
670 |     #[cfg(any(verus_keep_ghost, feature = "strict_provenance_atomic_ptr"))]
    |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: expected values for `feature` are: `alloc`, `default`, and `std`
    = help: consider adding `strict_provenance_atomic_ptr` as a feature in `Cargo.toml`
    = note: see <https://doc.rust-lang.org/nightly/rustc/check-cfg/cargo-specifics.html> for more information about checking conditional configuration
```

Reply:
Thank you!